Python/symtable.c

```diff
@@ -2536,11 +2536,22 @@ symtable_visit_expr(struct symtable *st, expr_ty e)
 static int
 symtable_visit_type_param_bound_or_default(
     struct symtable *st, expr_ty e, identifier name,
-    void *key, const char *ste_scope_info)
+    type_param_ty tp, const char *ste_scope_info)
 {
+    if (_PyUnicode_EqualToASCIIString(name, "__class__") ||
+        _PyUnicode_EqualToASCIIString(name, "__classdict__")) {
+
+        PyObject *error_msg = PyUnicode_FromFormat("reserved name '%U' can not be "
```

Comment:
```suggestion
        PyObject *error_msg = PyUnicode_FromFormat("reserved name '%U' cannot be "
```

Reply:
Believe it or not I did actually check before committing and they are both used in the codebase xD